### PR TITLE
Oembed fix

### PR DESCRIPTION
**Pull request by kak-tus**
Oembed support seems to be broken.
May be in case of running alerta in docker+postgresql. May be entirelly.

As example: /api/oembed endpoint accessible as /api/oembed, but not as /api/oembed.json.
Also: qb.from_params forms incorrect query to postgresql.

I am not a python programmer and don't know flask framework, so I fixed this not ideally.

After this fixes I've got oembed work in grafana
image

### CODE REVIEW

File: examples/oembed.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-  <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css" />
+  <link rel="stylesheet" href="http://netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css" />
```
Comment (vykulakov) [inline examples/oembed.html]: Why have you changed `//` to `http://` here and in other places of code? `//` looks like a more general approach that will work in a wider range of environments. For instance, I'm trying to access the `oembed.html` page bu HTTPs in my experiments.

File: examples/oembed.html [hunk 2]
```diff
@@ -13,22 +13,27 @@
 
 <div class="mobile-alerts">Loading...</div>
 
-<script src="//ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
-<script src="//netdna.bootstrapcdn.com/bootstrap/3.3.2/js/bootstrap.min.js"></script>
-<script src="http://api.alerta.io/embed.js"></script>
+<script src="http://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
+<script src="http://netdna.bootstrapcdn.com/bootstrap/3.3.2/js/bootstrap.min.js"></script>
+<script src="http://localhost:8080/api/embed.js"></script>
```
Comment (vykulakov) [inline examples/oembed.html]: Using the `/api` path prefix here is not correct because it will work only in your environment. Please, use more general URL like this: `//localhost:8080/embed.js`

File: examples/oembed.html [hunk 3]
```diff
@@ -13,22 +13,27 @@
...
 <script>
-$(function() {
+$(window).on('load', function () {
 
   // $.alerta.defaults.endpoint = 'http://localhost:8080';
   // $.alerta.defaults.key = 'demo-key';
   // or
   $.alerta.defaults = {
-      endpoint: 'http://api.alerta.io', // oEmbed endpoint becomes http://api.alerta.io/oembed.json
+      endpoint: 'http://localhost:8080/api', // oEmbed endpoint becomes http://localhost:8080/api/oembed
```
Comment (vykulakov) [inline examples/oembed.html]: See the comment about `/api` above.

File: examples/oembed.html (hunk 1 above)
Comment (kak-tus) [reply examples/oembed.html]: To get ability to user open this page locally and test with local instance of alerta.
Without this fix, user opens page with "file:///" protocol and external scripts tries to open with "file:///" protocol too.
If this page is not test page, but it is an example page - I will revert this fixes.

File: examples/oembed.html (hunk 3 above)
Comment (kak-tus) [reply examples/oembed.html]: Fixed.

File: examples/oembed.html (hunk 2 above)
Comment (kak-tus) [reply examples/oembed.html]: Fixed.

File: examples/oembed.html (hunk 1 above)
Comment (satterly) [reply examples/oembed.html]: Yes, please revert this change.

File: examples/oembed.html [hunk 4]
```diff
@@ -1,29 +1,44 @@
...
   // $.alerta.defaults.key = 'demo-key';
   // or
   $.alerta.defaults = {
-      endpoint: 'http://api.alerta.io', // oEmbed endpoint becomes http://api.alerta.io/oembed.json
+      endpoint: 'http://localhost:8080', // oEmbed endpoint becomes http://localhost:8080/oembed
       key: 'demo-key'
   };
 
-  $('.mobile-alerts').alerta('http://api.alerta.io/alerts/count?service=Mobile&status=open', {title:'Mobile Service'});
+  function renew() {
```
Comment (satterly) [inline examples/oembed.html]: Can you call this function `refresh()` instead of `renew()`? Thanks.

File: alerta/views/oembed.py [hunk 5]
```diff
@@ -9,28 +9,21 @@
...
-    if 'url' not in request.args or 'maxwidth' not in request.args \
-            or 'maxheight' not in request.args:
-        return jsonify(status='error', message='missing default parameters: url, maxwidth, maxheight'), 400
-
     try:
         url = request.args['url']
-        width = int(request.args['maxwidth'])
-        height = int(request.args['maxheight'])
-        title = request.args.get('title', 'Alerts')
+        title = request.args['title']
```
Comment (satterly) [inline alerta/views/oembed.py]: Why did you remove the default title? It will throw an exception here now if someone does not provide `title` as a URL param.

**Comment by satterly** [comment]
Thanks for the PR. It looks good except for a few minor changes. I'll be happy to merge this once you've addressed the concerns I've raised above. 👍

**Comment by vykulakov** [comment]
Looks like I've forgotten to submit my review earlier. My mistake, sorry.
